### PR TITLE
example-with-oc

### PR DESCRIPTION
**Pull request by mattiaerre**
# Description

- use **oc-purple-pluto** component in `example/index.html`

- remove snippet

- add `trackerTarget` and `painterContainer` parameters

- re-organise **oc-purple-pluto** component html

- use only `yarn start:example` to run the demo

- set the radius to 10 (need fine tuning)

- other small improvements

### CODE REVIEW

**Comment by mattiaerre** [comment]
will fix #3

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "bundle": "webpack",
     "copy": "cp lib/bundle.js oc-purple-pluto/public/javascripts/",
-    "start:example": "OUTPUT_PATH=example webpack-dev-server --content-base example/",
+    "start:example": "concurrently --kill-others 'yarn start:oc' 'yarn bundle && yarn copy && serve -p 8080 example'",
```
Comment (mattiaerre) [inline package.json:9]: I'm proud of myself 😅

File: src/make-model.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ const makeModel = (e) => { // eslint-disable-line arrow-body-style
     x: e.pageX,
     y: e.pageY,
     value: 1,
-    radius: 20
+    radius: 10
```
Comment (mattiaerre) [inline src/make-model.js:6]: I'd be cool to have different radius per different actions tracked (I will explain what I mean)

**Review by mariaathena — APPROVED** [review]
awesome work!!

**Pull request merged**